packages/stream-server-uws/package.js

```diff
@@ -0,0 +1,18 @@
+Package.describe({
+  summary: "Meteor's DDP stream server based on uWS",
+  version: '0.5.1',
+  documentation: 'README.md'
+});
+
+Package.onUse(function (api) {
+  api.use(['modules'], 'server');
```

Comment:
Let's just do `api.use("ecmascript")` here, to get full ES module support.

Reply:
Package `ecmascript` it is not ECMAScript modules for Node.js, it is the Babel.

After Babel compilation the result code is horrible.

I wrote the `stream-server-uws` package for high performance, and I don't want any possible interference from Babel. 

## use `modules` way

### Code

`stream_server_uws.js`

```js
const { RoutePolicy } = require('meteor/routepolicy');
const { WebApp } = require('meteor/webapp');
// ...
StreamServerUWS = class StreamServerUWS {
  // ...
}
```

### Result

`.meteor/local/build/programs/server/packages/stream-server-uws.js`

```js
const { RoutePolicy } = require('meteor/routepolicy');
const { WebApp } = require('meteor/webapp');
// ...
StreamServerUWS = class StreamServerUWS {
  // ...
}
```

## use `ecmascript` way

### Code

```js
const { RoutePolicy } = require('meteor/routepolicy');
const { WebApp } = require('meteor/webapp');
// ...
export class StreamServerUWS {
  // ...
}
```

### Result

`.meteor/local/build/programs/server/packages/stream-server-uws.js`

```js
const module1 = module;
module1.export({
  StreamServerUWS: () => StreamServerUWS
});
let RoutePolicy;
module1.watch(require("meteor/routepolicy"), {
  RoutePolicy(v) {
    RoutePolicy = v;
  }

}, 0);
let WebApp;
module1.watch(require("meteor/webapp"), {
  WebApp(v) {
    WebApp = v;
  }

}, 1);
// ...
class StreamServerUWS {
  // ...
}
```